docs/windows.md

```diff
@@ -61,86 +67,38 @@ shows the Antrea OVS bridge configuration on a Windows Node, and NetworkPolicy
 
-```powershell
-$InstallKubeProxy=$false # Only for kube-proxy mode
-$KubernetesVersion="v1.23.5"
-$KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
-$KubeletKubeconfigPath="C:/etc/kubernetes/kubelet.conf"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="C:/Users/Administrator/kubeproxy.conf" }
-```
-
-```powershell
-$TAG="v1.13.0"
-$KubernetesVersion="<KubernetesVersion>"
-$KubeConfig="<KubeconfigPath>"
-$KubeletKubeconfigPath="<KubeletKubeconfigPath>"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="<KubeProxyKubeconfigPath>" }
-$KubernetesHome="c:/k"
-$AntreaHome="c:/k/antrea"
-$KubeProxyLogPath="c:/var/log/kube-proxy"
-
-curl.exe -LO "https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Helper.psm1"
-Import-Module ./Helper.psm1
-
-Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
-New-KubeProxyServiceInterface
-
-New-DirectoryIfNotExist "${AntreaHome}/logs"
-New-DirectoryIfNotExist "${KubeProxyLogPath}"
-# Install kube-proxy service
-if ($InstallKubeProxy) { nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=${KubeProxyKubeconfigPath} --log-dir=${KubeProxyLogPath} --logtostderr=false --alsologtostderr" }
-nssm install antrea-agent "${AntreaHome}/bin/antrea-agent.exe" "--config=${AntreaHome}/etc/antrea-agent.conf --logtostderr=false --log_dir=${AntreaHome}/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
-
-nssm set antrea-agent DependOnService ovs-vswitchd
-if ($InstallKubeProxy) { nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd }
-nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
-
-if ($InstallKubeProxy) { Start-Service kube-proxy }
-Start-Service antrea-agent
-```
+### Prerequisites
 
-### Installation as a Pod (Docker/containerd based runtimes)
+* Create a Kubernetes cluster.
+* Obtain a Windows Server 2019 license (or higher) in order to configure the
+  Windows Nodes that will host Windows containers. And install the latest
+  Windows updates.
+* On each Windows Node, install the following:
+  - [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server)
+    with management tools. If your Nodes do not have the virtualization
+    capabilities required by Hyper-V, use the workaround described in the
+    [Knownissues](#known-issues) section.
+  - [containerd](https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=containerd#windows-server-1).
+
+### Installation as a Pod
+
+This installation method requires Antrea 1.10 or higher, and containerd 1.6 or
+higher (containerd 1.7 or higher is recommended). It relies on support for
+[Windows HostProcess containers](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/),
```

Comment:
In the windows-design.md, you referred the same link with context "Windows HostProcess Pod".

Reply:
Changed it to "Pod" here, but the K8s documentation for this feature uses both terminologies